### ISSUE TITLE
Check which command are necessary

### ISSUE THREAD

**Issue by eerison** [sonata-project/SonataPageBundle#1465]
When I was working in this PR: #1460 , I realized that some commands maybe are not used because there is possible problems, But should be good understand the reason to have those commands and improve the code and coverage with tests.

Commands that must be checked

- #1473 [MigrateBlockNameSettingCommand](https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/Command/MigrateBlockNameSettingCommand.php)
- #1473 [MigrateToJsonTypeCommand](https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/Command/MigrateToJsonTypeCommand.php)
- [x] [CreateBlockContainerCommand](https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/Command/CreateBlockContainerCommand.php#L62
) (I won't touch in this command for now, because I'm not sure if it's useful or not, we can handle this in other ticket if it's necessary)

**Comment by jordisala1991** [comment]
The migrate to commands maybe need to be removed. The Json one was to migrate a column from the old json type from sonata to json from doctrine (or something like that).

I removed that one from SonataMediaBundle. (we need to ensure the column types are correct)

**Comment by eerison** [comment]
Nice thanks for the update :)

**Comment by jordisala1991** [comment]
Going to the initial pr that introduces the command might help too.

**Comment by eerison** [comment]
Ok I was checking the commands and those migrations, and 
- [MigrateBlockNameSettingCommand](https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/Command/MigrateBlockNameSettingCommand.php) was added in this PR: #112, the current code from 3.x branch just works if use `code` instead of `name` in settings

- [MigrateToJsonTypeCommand](https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/Command/MigrateToJsonTypeCommand.php) the current code of 3.x requires the settings be `json` otherwise it'll crash the block, then you must use json.

and as @jordisala1991 said it was code to do some migration from old version both codes are 10 years old 😃 

- [CreateBlockContainerCommand](https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/Command/CreateBlockContainerCommand.php#L62) this command I won't touch because I'm not sure if it's useful or not, I saw that it's in the doc but well we do not used this in our project, who created this was @core23, maybe he can help us somehow here.


but for the first 2 commands I'll open a PR to remove them.
